### PR TITLE
Issue 166 randomly load projects home

### LINKED ISSUE #166: SPIKE - Can we randomly load resources to resources "We're getting started" tile on home page?
What we will learn: Currently, the "We're just getting started" tile on the home page loads the same four resources. Can we randomly load these resources to provide more visibility to others?

What we will have after card is complete: We will document how we can randomly load resources (if possible) and create a card to complete the work.

### PR DESCRIPTION
**Pull request by maharris1011**
uses the csc-airtable-api /projects route to pull random list of 4 projects to feature on the page every time the page is loaded.

uses vue.js for re-writing the DOM and axios for network call to the api.

see api definition at cantstopcbus/csc-airtable-api

fixes #166

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -43,18 +48,53 @@
...
+    }
 
-  $(function() {
-    $('img').addClass('img-fluid');
-  })
+    $(function() {
+      $('img').addClass('img-fluid');
+    })
 
+    var cantstopcbus = new Vue({
```
Comment (seanlawrenz) [inline _includes/footer.html:67]: You're going to make me learn Vue aren't you?

File: _includes/footer.html [hunk 2]
```diff
@@ -43,18 +48,53 @@
...
+          .then((response) => {
+            let shuffle = (array) => {
+              for(var i = array.length - 1; i> 0; i--) {
+                const j = Math.floor(Math.random() * i)
+                const temp = array[i]
+                array[i] = array[j]
+                array[j] = temp
+              }
+              return array
+            }
```
Comment (seanlawrenz) [inline _includes/footer.html:84]: Should there be a check that response was ok? I don't remember if `axios` does a successful return on a 404.

File: _includes/footer.html (hunk 1 above)
Comment (maharris1011) [reply _includes/footer.html]: absolutely! :-)

File: _includes/footer.html (hunk 2 above)
Comment (maharris1011) [reply _includes/footer.html]: good question. i'll check.

**Comment by maharris1011** [comment]
@seanlawrenz @jranta @CoreySchnedl - would you all please take another look at the code & make sure you're OK with everything before I go live? I would hate for there to be a problem, or just ugly/unmaintainable Javascript code.

**Pull request merged**